thorlcr/activities/thdiskbase.ipp

```diff
@@ -32,12 +32,15 @@ protected:
     Owned<CSlavePartMapping> mapping;
     IHash *hash;
     StringAttr fileName;
+    std::vector<OwnedPtr<CThorStatsCollection>> subFileStats;
 
 public:
     CDiskReadMasterBase(CMasterGraphElement *info);
     virtual void init();
     virtual void serializeSlaveData(MemoryBuffer &dst, unsigned slave);
+    virtual void done() override;
```

Comment:
trivial: good change, for consistency would be good if other 'override' were added at same time.